docker-compose.yml

```diff
@@ -90,17 +90,10 @@ services:
           - ES_ADDRESS
       build:
           context: platformtools
+          platforms:
+              - "linux/amd64"
```

Comment:
Why is this needed? And why only for this service?

Reply:
This must not be here actually as it's irrelevant to this branch. I made this change to make this container compatible with Apple M1/M2 CPUs. But need to check for backward compatibility on a Linux machine. I will open a separate issue for this.

Reply:
Issue is here: https://github.com/mdenet/educationplatform-docker/issues/65